spec/spec_helper.rb

```diff
@@ -22,6 +22,9 @@
 require_relative 'support/user_sign_in_helper.rb'
 include UserSignIn
 
+require_relative 'support/bucket_list_sign_in_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.